CMakeLists.txt

```diff
@@ -221,6 +221,14 @@ rocm_enable_cppcheck(
 enable_testing()
 
 include(ROCMCreatePackage)
+rocm_create_package(
+    NAME MIGraphX
+    DESCRIPTION "AMD's graph optimizer"
+    MAINTAINER "Paul Fultz II <paul.fultz@amd.com>"
```

Comment:
Should we change the email on this to amd-maintainer?

Reply:
no, just backing out.  No other changes.